src/org/camunda/latera/bss/connectors/MailSender.groovy

```diff
@@ -172,19 +172,30 @@ class MailSender implements AutoCloseable {
     return this
   }
 
-  MailSender addFile(CharSequence name, Object datasource) {
+  MailSender addHTMLPart(CharSequence body) {
+    MimeBodyPart part = new MimeBodyPart()
+    part.setText(body.toString(), 'utf-8', 'html')
+    this.multipart.addBodyPart(part)
+    return this
+  }
+
+  MailSender addFile(CharSequence name, Object datasource, CharSequence content_disposition = 'inline') {
```

Comment:
In Java and Groovy variables are named in camelCase, not snake_case.

Reply:
Default value of contentDisposition should be 'attachment'.
Also I think that Boolean should be used here instead.

Reply:
This is not boolean. This is value of `Content-Disposition`.
If you want to, I can add two methods – `addFileInline`, `addFileAttachment`.